components/tooltip/README.md

```diff
@@ -0,0 +1,122 @@
+| on        | String    | click   | `click`, ...                     | Specify the event which will trigger the show method of the tooltip                              |
+| off       | String    | click   | `click`, ...                     | Specify the event which will trigger the hide method of the tooltip                              |
+| position  | String    | top     | `top`, `bottom`, `left`, `right` | Specify the position of the tooltip relative to the element specified using the `target` attribute. |
+
+
+If you wish to import the modules using JavaScript you can remove the script tags
+which import the components and the tooltip from the node_modules folder and import them like this:
+
+~~~~{.js}
+import components from 'coherent-gameface-components';
+import Tooltip from 'coherent-gameface-tooltip';
+~~~~
+
+Note that this approach requires a module bundler like [Webpack](https://webpack.js.org/) or [Rollup](https://rollupjs.org/guide/en/) to resolve the
+modules from the node_modules folder. Alternatively you can import them directly from node_modules:
+
+~~~~{.js}
+import components from './node_modules/coherent-gameface-components/umd/components.production.min.js';
+import Tooltip from './node_modules/coherent-gameface-tooltip/umd/tooltip.production.min.js';
+~~~~
+
+## Usage with CJS modules:
+
+* Import the components library:
+
+~~~~{.js}
+const components = require('coherent-gameface-components');
+const Tooltip = require('coherent-gameface-tooltip');
+~~~~
+
+The CommonJS(CJS) modules are used in a NodeJS environment, be sure to use a module
+bundler in order to be use them in a browser.
+
+
+Specifying the content
+=========================
+
+Use the message slot to specify the message of the tooltip.
+
+~~~~{.html}
+<gameface-tooltip target=".container" on="mouseenter" off="mouseleave" position="left">
+    <div slot="message">Message on left</div>
+</gameface-tooltip>
+~~~~
+
+You can also put more complex content like interactive controls such as buttons:
+
+~~~~{.html}
+<gameface-tooltip id="tutorial" target=".container" on="mouseenter" position="top" off="click">
+    <div slot="message">
+        <div class="msg-container">
+            <div class="title">Wellcome!</div>
+            <div class="text">
+                Hello, this is your first tutorial.
+                Click the next button to continue.
+            </div>
+            <div class="button">Next</div>
+        </div>
+    </div>
+</gameface-tooltip>
+~~~~
+
+You can add more flexibility using JavaScript. In the above example the tooltip will be closed on click anywhere inside it. If you wish to close it only if the customer clicks for the example on the `Next` button you can manually attach a listener and call the `hide` method of the tooltip:
+
+~~~~{.js}
+document.querySelector('.button').addEventListener('click', (e) => {
+    document.querySelector('#tutorial').hide();
+});
+~~~~

```

Comment:
https://github.com/CoherentLabs/GameUIComponents/pull/52#discussion_r754945095

An example would be if there is a "close" button but the text within the tooltip to be selected/copied. This means that the whole tooltip will not be a "close" button anymore.

Here you can hover over -> tooltip gets visualized -> you can click and copy the text https://www.w3schools.com/css/css_tooltip.asp